.gitattributes

```diff
@@ -64,11 +64,6 @@
 *.bat text eol=crlf
 
 # Custom for Visual Studio
-*.csproj merge=union
-*.vbproj merge=union
-*.fsproj merge=union
-*.dbproj merge=union 
-*.sln text eol=crlf merge=union
```

Comment:
Where did these come from in the first place?

Reply:
The previous repostandards file add them. Some early template from GitHub I think included them.